File: test/fixtures/output/node/unirest/cookies.js
const unirest = require("unirest");

const req = unirest("GET", "https://httpbin.org/cookies");

const CookieJar = unirest.jar();
CookieJar.add("foo=bar","https://httpbin.org/cookies");
CookieJar.add("bar=baz","https://httpbin.org/cookies");
req.jar(CookieJar);

req.end(function (res) {
  if (res.error) throw new Error(res.error);

  console.log(res.body);
});

File: src/targets/c/libcurl/fixtures/cookies.c
CURL *hnd = curl_easy_init();

curl_easy_setopt(hnd, CURLOPT_CUSTOMREQUEST, "GET");
curl_easy_setopt(hnd, CURLOPT_URL, "https://httpbin.org/cookies");

curl_easy_setopt(hnd, CURLOPT_COOKIE, "foo=bar; bar=baz");

CURLcode ret = curl_easy_perform(hnd);